Project Files/Calculator/Calculator.java

```diff
@@ -41,7 +41,7 @@ public Calculator(double currAlk, double currChl, double currCyAcid, double curr
         this.currHard = currHard;
         this.currPh = currPh;
         this.pool_volume = pool_volume;
-        calc_volumeFactor();
+        calcVolumeFactor();
```

Comment:
As you can see here, I went with a standard camel case style instead of doing the underscores across the board. I was first trying to see if the underscores were easier read, but I think it might just be better to stick with the camel case. Anybody else have an opinion on that?